### PR TITLE
Introduction to json tutorial c# language update

### PR DESCRIPTION
**Pull request by Srainyyyy**
I have added C# OOP code to this tutorial, below is a screenshot of the code test：
image

image

clear game_data.json
image

### CODE REVIEW

**Review by s223126445 — APPROVED** [review]
Tutorial Review - Introduction to json tutorial c# language update
 Good work, both the codes are working fine.But please update the OOP code similar to the top-level statement.
Screenshots of code test:
Opp_test1
OOP_test2

File: src/content/docs/guides/JSON/0-json_intro.mdx [hunk 1]
```diff
@@ -89,6 +92,39 @@ This command creates sub-folders for each type of resource. One of these is name
...
+  ```csharp
+using SplashKitSDK;
+
+namespace GameDataManagement
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Define the file path
```
Comment (breezy-codes) [inline src/content/docs/guides/JSON/0-json_intro.mdx:106]: I would remove these comments in the OOP version as there is no comments in the C++ or the C# top level statement version. So the comments within this OOP one look really out of place.

File: src/content/docs/guides/JSON/0-json_intro.mdx [hunk 2]
```diff
@@ -191,7 +230,45 @@ else
...
+```csharp
+using SplashKitSDK;
+
+namespace GameDataManagement
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Define the file path
```
Comment (breezy-codes) [inline src/content/docs/guides/JSON/0-json_intro.mdx:245]: Similar in this one, the comments look out of place when comparing to the C++ and C# top level statement versions.

**Review by breezy-codes — CHANGES_REQUESTED** [review]
The code runs correctly and looks good, although the comments within the code need to be removed so that they match with how the code is in the C++ and C# top level versions.

**Review by breezy-codes — APPROVED** [review]
Looks great now, everything works correctly.

**Pull request merged**